editor/themes/editor_theme_manager.cpp

```diff
@@ -1679,7 +1680,12 @@ void EditorThemeManager::_populate_editor_styles(const Ref<EditorTheme> &p_theme
 		p_theme->set_stylebox("BottomPanel", EditorStringName(EditorStyles), style_bottom_panel);
 
 		// Main menu.
-		p_theme->set_stylebox("MenuHover", EditorStringName(EditorStyles), p_config.button_style_hover);
+		Ref<StyleBoxFlat> style_widget_transparent = p_config.button_style->duplicate();
+		style_widget_transparent->set_bg_color(Color(1, 1, 1, 0));
+		style_widget_transparent->set_border_width_all(0);
+		Ref<StyleBoxFlat> style_widget_hover = p_config.button_style_hover->duplicate();
```

Comment:
You should use more specific and descriptive names, "widget" means nothing in this context. `menu_transparent_style`, `menu_hover_style`, or something similar, would be more fitting.